.github/workflows/integration-tests.yml

```diff
@@ -776,7 +777,7 @@ jobs:
 
   testnet-smoke-tests-notify:
     name: Live Testnet Start Slack Thread
-    if: ${{ github.event_name == 'schedule' || (github.event_name == 'push' && startsWith(github.ref, 'refs/tags/')) }} ## Only run live tests on new tags and nightly
+    if: success() || failure()
```

Comment:
is it not similar to always() ?

Reply:
`always()` will run every time, even if previous steps are **skipped**. `success() || failure()` will run whether previous steps failed or succeeded, but not skipped.